lib/takeaway.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/dish'
+require './lib/order'
+
+class Takeaway
+  attr_reader :menu_list
+  
+  def initialize(name)
+    @name = name
+    @menu_list = []
+    load_menu(name)
+  end
+
+  def show_menu
+    dishes = []
+    @menu_list.each { |dish| dishes << "#{dish.name} - #{dish.price}" } 
+    dishes.join(', ')
+  end
+
+  def make_order
+    Order.new(@menu_list)
```

Comment:
This is a hard dependency between your takeaway class and your Order class. Could you use dependency injection here? If it is tricky, maybe you need to think about the structure of your classes. For example, does an order need to know what menu was used to make it?